packages/date-time/src/components/Calendar/examples/Calendar.Inline.Example.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { DefaultButton } from 'office-ui-fabric-react';
-import { addDays, getDateRangeArray } from 'office-ui-fabric-react/lib/utilities/dateMath/DateMath';
+import { addDays, getDateRangeArray } from '../../../utilities/dateMath/DateMath';
```

Comment:
Longer term, do you plan on having the date utilities in this package diverge from the ones in OUFR? If not, I think it would be better to have the utilities in this package just re-export the ones from OUFR--I actually was just about to make that change in the Fabric 7 branch in another PR. https://github.com/OfficeDev/office-ui-fabric-react/pull/9253/files#diff-8eae178ff00b175d4b7ebc9c199b5c8e

Reply:
I think we're planning on removing the date utilities from OUFR entirely in favor of these ones in date-time

Reply:
The trouble with removing the utilities from OUFR is that then when the old DatePicker and Calendar move to the legacy package (in that PR), they'll need to have a duplicate copy of the utilities. At least to me, it seemed simpler to leave the utilities in OUFR for now so both date-time and legacy can depend on them (and date-time can re-export them), but I'm open to suggestions. Also we might need to give more warning if we were going to remove the utilities from OUFR.

Reply:
In that case you should be able to do `import { ... } from '@uifabric/date-time'`.

Reply:
I'm open to leaving them there, I don't think we've discussed those utilities in depth. I'll see about doing the re-export so long as I'm here in this PR then

Reply:
Having looked at your PR, it looks like you've taken a more thorough approach to this, so I'll revert this part of the change and let yours do it.

Reply:
changing it to '@uifabric/date-time' comes back with a failure when running tests:

 RUNS  src/components/DatePicker/DatePicker.test.tsx
  ● Test suite failed to runce...

    Cannot find module '@uifabric/date-time' from 'Calendar.base.tsx'

       9 |   ICalendarStyles
      10 | } from './Calendar.types';
    > 11 | import { DayOfWeek, FirstWeekOfYear, DateRangeType } from '@uifabric/date-time';
         | ^
      12 | import { CalendarDay } from './CalendarDay/CalendarDay';
      13 | import { CalendarMonth } from './CalendarMonth/CalendarMonth';
      14 | import { ICalendarDay } from './CalendarDay/CalendarDay.types';

      at Resolver.resolveModule (../../common/temp/node_modules/.registry.npmjs.org/jest-resolve/23.6.0/node_modules/jest-resolve/build/index.js:221:17)
      at Object.<anonymous> (src/components/Calendar/Calendar.base.tsx:11:1)


I think I'll try to leave that for the next PR, and leave the reference to OUFR for now

Reply:
You need an additional line in the project's jest.config.js--look at OUFR's config for an example.